### PR TITLE
[TOOLS-2292] Add EON Connection Details

### PR DESCRIPTION
**Pull request by jmontesdeocanuez**
Jira ticket: [TOOLS-2292 Add EON Connection Details](https://horizenlabs.atlassian.net/browse/TOOLS-2292)

In this PR we add the EON connection details to the docs.

image
image

### CODE REVIEW

File: docs/02-connect/02-connect_your_wallet.md [hunk 1]
```diff
@@ -37,14 +37,22 @@ Adding a network example:
...
+
+Once you have specified the RPC endpoint and Chain ID to make a connection, a notification appears stating, **“EON Mainnet was successfully added”**.
 
 ##### Gobi Testnet
 
       Network name:  Gobi Testnet
       New RPC URL:  https://gobi-testnet.horizenlabs.io/ethv1
       Chain ID: 1663 
-      Currency symbol: ZEN
+      Currency symbol: tZEN
```
Comment (taijonsmith) [inline docs/02-connect/02-connect_your_wallet.md:55]: @jmontesdeocanuez @yurikoinaba can we confirm if the testnet currency should be `tZEN` or `TZEN`? The EON [dashboard](https://eon.horizen.io/status/gobi) and most vars I've seen uses `TZEN`

Comment (yurikoinaba) [reply docs/02-connect/02-connect_your_wallet.md:55]: Let's keep it consistent w other placements in the documentation

Comment (jmontesdeocanuez) [reply docs/02-connect/02-connect_your_wallet.md:55]: image
image

Actually, right now we're not consistent, we are using both of 'tZEN' and 'TZEN' across the site. Should I update all to be the same? In that case, should them be 'tZEN' or 'TZEN' @yurikoinaba

Comment (yurikoinaba) [reply docs/02-connect/02-connect_your_wallet.md:55]: @jmontesdeocanuez Ok in that case, let's make it `TZEN` everywhere. Thank you both

**Review by yurikoinaba — APPROVED** [review]
lgtm. Let's wait to merge until the Node Gateway (RPC URLs work properly)

**Review by taijonsmith — APPROVED** [review]

File: docs/02-connect/02-connect_your_wallet.md [hunk 2]
```diff
@@ -37,14 +37,22 @@ Adding a network example:
 
 ![alt_text](/img/docs/get-started/metamask-add-network2.png)
 
+##### EON Mainnet
```
Comment (yurikoinaba) [inline docs/02-connect/02-connect_your_wallet.md:40]: Didn't notice this earlier. Let's make it `Horizen EON`.

Comment (yurikoinaba) [reply docs/02-connect/02-connect_your_wallet.md:40]: There's two other places in this file

**Pull request merged**